mia/__main__.py

```diff
@@ -0,0 +1,120 @@
+"""
+MIA - Mission Impossible: Hardening Android for Security and Privacy
+
+This project is a attempting to streamline the process of following Mike Perry's
+Android hardening tutorial on the Tor blog:
+    https://blog.torproject.org/blog/mission-impossible-hardening-android-security-and-privacy
+
+Please keep in mind that this is experimental, and may not be functional at any
+given moment. Also, it will likely wipe your Android device, and this is by
+design!
+
+Usage:
+    mia [options] <command> [<command_args_and_opts>...]
+    mia --help
+
+Global options:
+    --debug     Spew out debug information.
+    --verbose   Spew out even more information than normal.
+    --quiet     Restrict output to warnings and errors.
+    -h, --help  Show this screen.
+    --version   Show version.
+
+Available commands:
+    build       Build an update.zip file.
+    clean       Cleanup the current workspace.
+    definition  Create and configure a definition for a new update.zip file
+                based on existing templates.
+    help        Display detailed information about a certain command.
+    install     Install the OS and the built update.zip file onto the device.
+
+See 'mia help <command>' for more information on a specific command.
+
+"""
+
+# Get the path of the mission-impossible-android project.
+import os
```

Comment:
imports should always be at the top: https://www.python.org/dev/peps/pep-0008/#imports